.github/workflows/version_bump.yml

```diff
@@ -0,0 +1,64 @@
+name: Version Bump and Release Branch Creation
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_bump:
+        description: 'Version bump type (major, minor, patch)'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+      release_description:
+        description: 'Short description of the release (motivation, changes, tests)'
+        required: true
+
+jobs:
+  version_bump:
+    runs-on: ubuntu-24.04
+
+    steps:
+
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16'
+
+    - name: Install dependencies
```

Comment:
for this step and the above, better to use `prepare` action

Reply:
done